### PR TITLE
adding time varying Kw

### PR DESCRIPTION
**Pull request by jread-usgs**
Haven't build the viz, but this should work for you @lindsayplatt

### CODE REVIEW

**Comment by jread-usgs** [comment]
@lindsayplatt can you take another look? I've now built the htmls and made some small changes to your code - removing the `I()` blocks - wondering if those needed to be used?

File: 4_params_munge/src/munge_secchi_model_data.R [hunk 1]
```diff
@@ -1,19 +1,18 @@
-munge_daily_secchi <- function(in_ind, out_ind, dow_crosswalk_ind) {
+munge_daily_secchi <- function(out_ind, kw_files_zip_ind) {
 
-  dowfile <- sc_retrieve(dow_crosswalk_ind, remake_file = '1_crosswalk_fetch.yml')
-  dow2nhd <- get(load(dowfile))
-  dow2nhd$DOW <- as.character(dow2nhd$dowlknum)
-  dow2nhd <- distinct(dow2nhd)
+  cwd <- getwd()
+  on.exit(setwd(cwd))
```
Comment (lindsayplatt) [inline 4_params_munge/src/munge_secchi_model_data.R]: It doesn't look like you ever change the current working directory. Are these two steps necessary?

File: 4_params_munge/src/munge_secchi_model_data.R [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
-           DOW = as.character(DOW)) %>%
-    left_join(dplyr::select(dow2nhd, site_id, DOW) , by = 'DOW') %>%
-    dplyr::select(-Secchi_ft, -DOW)
+  kw_files_zip <- sc_retrieve(kw_files_zip_ind)
+  zip_dir <- tempdir()
+  kw_files <- unzip(zipfile = kw_files_zip, overwrite = TRUE, exdir = zip_dir)
+
+  purrr::map(kw_files, function(x){
+    read_csv(x, col_types = 'Dd') %>% mutate(site_id = str_remove(basename(x), pattern = '_kw.csv'))
+  }) %>% purrr::reduce(bind_rows) %>%
```
Comment (lindsayplatt) [inline 4_params_munge/src/munge_secchi_model_data.R]: I really need to learn how to use `purrr::map` and `purrr::reduce`

File: 8_viz/src/merge_lake_data.R [hunk 3]
```diff
@@ -108,9 +108,11 @@ merge_lake_data <- function(out_ind, temp_data_fl, lake_depth_ind, lake_names_in
 
 # Summarizing the information related to TOHA models
 summarize_MN_toha_lake_data <- function(out_ind, mndow_xwalk_ind, lake_summary_ind,
-                                        walleye_count_data, plant_priority_data) {
+                                        walleye_count_ind, plant_priority_ind) {
 
   mndow_xwalk <- readRDS(sc_retrieve(mndow_xwalk_ind))
+  walleye_count_data <- read_csv(sc_retrieve(walleye_count_ind))
+  plant_priority_data <- read_csv(sc_retrieve(plant_priority_ind))
```
Comment (lindsayplatt) [inline 8_viz/src/merge_lake_data.R]: thanks for moving here

**Comment by lindsayplatt** [comment]
FYI, after merge was able to pull master down and build the new data in 8_viz with no issues 👍 

```
scmake("8_viz/inout/lakes_summary.feather.ind", remake_file = "8_viz.yml")
Starting build at 2020-03-03 12:03:01
<  MAKE > 8_viz/inout/lakes_summary.feather.ind
[    OK ] 8_viz/inout/lakes_summary.feather.ind
Finished build at 2020-03-03 12:03:01
Build completed in 0.00 minutes
```

**Pull request merged**